data/json/items/vehicle_parts.json

```diff
@@ -211,6 +211,21 @@
     "category": "veh_parts",
     "price": 20000
   },
+  {
+    "type": "GENERIC",
+    "id": "workbench",
+    "name": "workbench",
+    "description": "A crude wooden table.",
```

Comment:
Wooden table with only steel as a material.

Reply:
Whoops, forgot to replace that, thanks